apps/dashboard/app/apps/manifest.rb

```diff
@@ -61,10 +61,13 @@ def self.load_from_string(yaml)
   # @option opts [String] :role Dashboard categorization
   # @option opts [String] :url An optional redirect URL
   # @option opts [Hash]   :metadata An optional hash of key value pairs
+  attr_reader :name, :description
   def initialize(opts)
     raise InvalidContentError.new unless(opts && opts.respond_to?(:to_h))
 
     @manifest_options = opts.to_h.with_indifferent_access
+    #@name = opts[:name]
+    #@description = opts[:description]
```

Comment:
That reject should probably be here. Seems like it would cover merge and maybe load code paths?

Also be sure to remove that `attr_reader` above.

Reply:
Bah, can't believe I left that there. Yeah I'll yank it and move the reject up here and yank the sanitze_manifest from save.